src/utils/webhook.ts

```diff
@@ -4,6 +4,22 @@ import express, { json } from 'express'
 
 export const sleep = (ms: number) => new Promise((resolve) => setTimeout(resolve, ms))
```

Comment:
Are we still using this `sleep` function, should we keep it here?

Reply:
it is still used by the `waitForHook` function. We could stop the export, but it could be a useful export for the future so I woulnd't worry about it